.github/workflows/nightly.yml

```diff
@@ -16,7 +20,7 @@ jobs:
     with:
       version: ${{ inputs.version || 'now' }}
       default_modifier: "-gardener_prod"
-      use_kms: true
+      use_kms: ${{ github.event.inputs.use_kms == 'true' }}
```

Comment:
this is broken, the default key for an input **only** applies if it is actually run with workflow_dispatch, in the on schedule case this will always be empty => breaking kms support for nightly builds